tests/utils/utils.go

```diff
@@ -700,3 +699,15 @@ func GetTwoSubnets(network interfaces.Network) (
 	Expect(len(subnets)).Should(BeNumerically(">=", 2))
 	return subnets[0], subnets[1]
 }
+
+func CalculateMessageID(
```

Comment:
I don't think we need this extra utils function and can call the bidnings functions directly in test

Reply:
Unless anyone feels strongly, I'm in favor of keeping it. I think it is general enough that it is worth having as a utility, and that way it's less error prone because `Expect(err).Should(BeNil())` doesn't need to be duplicated everywhere.